docs/modelserving/v1beta1/onnx/README.md

```diff
@@ -0,0 +1,42 @@
+
+# Deploy InferenceService with ONNX model
+## Setup
+1. Your ~/.kube/config should point to a cluster with [KServe installed](https://github.com/kserve/kserve#installation).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
+
+## Create the InferenceService
+Apply the CRD
+```
+kubectl apply -f onnx.yaml 
```

Comment:
We can directly add the yaml on the doc without having to get the yaml from repo